llm_studio/app_utils/sections/chat.py

```diff
@@ -67,6 +69,14 @@ async def chat_tab(q: Q, load_model=True):
         assert q.client["experiment/display/chat/tokenizer"] is not None
         initial_message = "Chat History cleaned. How can I help you?"
 
+    # Load validation dataframe and texts
+    validation_dataframe = get_prediction_dataframe(cfg.output_directory)
+    if cfg.dataset.parent_id_column != "None":
+        validation_dataframe = validation_dataframe.loc[
+            validation_dataframe[cfg.dataset.parent_id_column] != None
```

Comment:
To fix the flake8 style check

```suggestion
            validation_dataframe[cfg.dataset.parent_id_column] is not None
```